sklearn/compose/_column_transformer.py

```diff
@@ -97,6 +101,11 @@ class ColumnTransformer(_BaseComposition, TransformerMixin):
         Keys are transformer names and values are the fitted transformer
         objects.
 
+    remainder_transformers_ : list or None
```

Comment:
why plural? that's only one, right? at least according to the docstring for ``remainder``.